src/Alerts/Mapper.ts

```diff
@@ -0,0 +1,17 @@
+import { Alert } from './Alert';
+
+export const mapAlertsApiResponseToAlerts = (alerts: any[]): Alert[] => alerts.map((alert: any) => ({
+	id: alert.id,
+	externalId: alert.external_id,
+	type: alert.type,
+	severity: alert.severity,
+	affectedArea: alert.affected_area as GeoJSON.GeoJsonObject,
+	perex: alert.perex,
+	state: alert.state,
+	summary: alert.summary,
+	origin: alert.origin,
+	provider: alert.provider,
+	providerUrl: alert.providerUrl,
```

Comment:
this is inconsistent case... But bug is on API :( Im gonna fix it on API right now.